sled-agent/src/instance_manager.rs

```diff
@@ -379,6 +285,10 @@ impl InstanceManager {
             .map_err(|_| Error::FailedSendInstanceManagerClosed)?;
         rx.await?
     }
+
```

Comment:
Damn, forgot to add back the comment.

Reply:
Fixed in [bbbe079](https://github.com/oxidecomputer/omicron/pull/5124/commits/bbbe079c12dd636b0f52c4e71b321ae044f643e4)